src/main/java/org/codewars/Menu.java

```diff
@@ -17,7 +17,13 @@ public void setUserById(int userID) {
     }
 
     public void showUsersList() {
+        System.out.println("Available users:");
 
+        for (int i = 1; i <= Users.values().length; i++){
+            System.out.format("%2d | %s%n", i, Users.getById(i).getName());
+        }
+
+        System.out.println("Choose the appropriate user");
```

Comment:
this print has no sence bc it shoul be in setUser or showUserInfo

Reply:
I've changed